frontend/src/components/MapView/Layers/AnticipatoryActionStormLayer/AAStormDatePopup/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Popup } from 'react-map-gl/maplibre';
+import _React from 'react';
+import { FeatureCollection, Point } from 'geojson';
+import { getHours, getDate } from 'date-fns';
+import { createStyles, makeStyles, Typography } from '@material-ui/core';
+import { getDateInUTC } from '../utils';
+
+function AAStormDatePopup({ timeSeries }: PopupProps) {
+  const classes = useStyles();
+
+  function is6AM(time: string) {
```

Comment:
What's the deal with the 6am test?

Reply:
initially I thought the tooltip was only displayed at 6am. Then I started the dscussion of that topic. And now I need to implement the new plan. I will reshape this. 

You know it is a draft PR. I misunderstood that you might have a look at it despite it being a draft and I did not anticipate that you might also push stuff as well (so push --force is excluded now, and rebase probably too) I will keep that in mind before pushing more stuff. :-)